### ISSUE TITLE
How to train resnet18 to the best accuracy?

### ISSUE THREAD

**Issue by zerolxf** [pytorch/vision#1166]
I recently did a simple experiment, training cifar10 with resnet18( torchvision.models), but I can't  achieve the desired accuracy(93%).

I found a GitHub repository where the example can be trained to 93% accuracy, [pytorch-cifar](https://github.com/kuangliu/pytorch-cifar). But his implementation is different from torchvision.models.resnet18, The difference may be [here](https://github.com/kuangliu/pytorch-cifar/issues/91#issuecomment-514933998).  

Is there any example to teach us how to train cifar10 with resnet18 to the optimal precision?

**Comment by fmassa** [comment]
The models in `torchvision` are tailored for ImageNet-sized images (224x224).
We do not provide smaller versions of the models for 32x32 sized images.

Here is an example on how to train wide residual networks (a variant of resnet) on Cifar, reaching top performances https://github.com/szagoruyko/wide-residual-networks/tree/master/pytorch
